portfolio/src/main/webapp/script.js

```diff
@@ -30,7 +30,42 @@ function findZoe() {
   const location = locations[Math.floor(Math.random() * locations.length)];
 
   // Add it to the page.
-  const link = document.getElementById("link");
+  const link = document.getElementById('link');
   link.innerText = location;
   link.href = locationToLink[location];
 }
+
+function grow() {
+  const srcToAlt = {
+    'images/basil1.jpg': 'Basil Day 0',
+    'images/basil2.jpg': 'Basil Day 10',
+    'images/basil3.jpg': 'Basil Day 20',
+    'images/basil4.jpg': 'Basil Day 30',
+    'images/basil5.jpg': 'Basil Day 40',
+    'images/basil6.jpg': 'Basil Day 60',
+  };
+  const image = document.getElementById('basil');
+  console.log(`Let's grow some basil`);
+
+  // Recursively update images every second
```

Comment:
Curious on why you made this recursive. Was it because you want it to try it out using recursion or was it because the combination of a for loop and the timeout function didn't do what you expect it to do?

Reply:
The second one. I tried for loop first and it did not work as I expected, because setTimeout immediately returns and execute the following code. 

I think there should be a smarter way to achieve the same outcome with a for loop and setTimeout, but I haven't figured out how.

Reply:
Actually I find a way to do this. See the new branch: #4

Reply:
I actually prefer your solution - using the functions setInterval and clearInterval looks more native in JS. Thanks for the suggestion! Do you mind if I add your piece to the branch?

Reply:
Sure. You could either make this change in the other PR, merge it into this one, and then we can approve this PR